bitnami/fluentd/README.md

```diff
@@ -237,11 +239,50 @@ This chart includes a `values-production.yaml` file where you can find some para
 + forwarder.rbac.pspEnabled: true
 ```
 
+- Enable forwarding logs to another service like ElasticSearch
+```diff
+-  aggregator.extraConfig: |
+-    # Send all logs to the standard output
+-    <match **>
+-      @type stdout
+-    </match>
++  aggregator.extraConfig: |
++   # Send all logs to elasticsearch 
++    <match **>
++      @type elasticsearch
++      host elasticsearch-master-headless.default.svc.cluster.local
++      port 9200
```

Comment:
Wouldn't it be better if this is not hardcoded? In the configmap.yaml example, we use env-vars. What do you think?

Reply:
Oh i didn't notice that. I will add it. Thank you.